library/dnet.c

```diff
@@ -1119,9 +1112,6 @@ int dnet_process_cmd_raw(struct dnet_backend_io *backend, struct dnet_net_state
 
 	err = dnet_send_ack(st, cmd, err, recursive);
 
-	if (!(cmd->flags & DNET_FLAGS_NOLOCK))
-		dnet_opunlock(n, &cmd->id);
-
```

Comment:
Any command processed by dnet_process_cmd_raw() came from i/o thread pool reactor: dnet_io_process(). dnet_io_process() use dnet_pop_request() to take *always* non locked key.